### PR TITLE
Add FAGCN model

### PR DESCRIPTION
**Pull request by Kaka23333**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

File: gammagl/models/fagcn.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        self.num_layers = num_layers
+        self.relu = nn.ReLU()
+        self.dropout = nn.Dropout(drop_rate)
+
+        self.mid_layers = nn.Sequential()
+        for i in range(num_layers):
+            self.mid_layers.append(FAGCNConv(src_degree=src_degree, dst_degree=dst_degree,
+                                             hidden_dim=hidden_dim, drop_rate=drop_rate))
+
+        init = nn.initializers.XavierNormal(seed=1.414)
```
Comment (clearhanhui) [inline gammagl/models/fagcn.py:45]: seed，同上

File: examples/fagcn/readme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+Results
+-------
+```bash
+TL_BACKEND="tensorflow" python fagcn_trainer.py --dataset cora --lr 0.01 --l2_coef 0.0005 --drop_rate 0.4 --hidden_dim 16 --eps 0.3 --num_layers 5
+TL_BACKEND="tensorflow" python fagcn_trainer.py --dataset citeseer --lr 0.01 --l2_coef 0.0005 --drop_rate 0.4 --hidden_dim 16 --eps 0.4 --num_layers 4
+TL_BACKEND="tensorflow" python fagcn_trainer.py --dataset pubmed --lr 0.01 --l2_coef 0.001 --drop_rate 0.4 --hidden_dim 16 --eps 0.1 --num_layers 6
+```
+| Dataset  | Paper    | Our(tf)  |
```
Comment (clearhanhui) [inline examples/fagcn/readme.md:36]: 测一下paddle后端的性能
